src/qibojit/backends/__init__.py

```diff
@@ -1,2 +1,44 @@
+from qibo.config import raise_error
+
 from qibojit.backends.cpu import NumbaBackend
 from qibojit.backends.gpu import CupyBackend, CuQuantumBackend
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibojit backends."""
+
+    def __init__(self):
+        self.platforms = ("numba", "cupy", "cuquantum")
+
+    def load(self, platform: str) -> NumbaBackend | CupyBackend | CuQuantumBackend:
```

Comment:
It could be useful to give it a name:
```suggestion
    def load(self, platform: str) -> QibojitBackend:
```
and this top-level:
```py
NumbaBackend | CupyBackend | CuQuantumBackend
```